copier/main.py

```diff
@@ -651,7 +651,8 @@ def run_copy(self) -> None:
             if not self.quiet:
                 # TODO Unify printing tools
                 print("")  # padding space
-            self._execute_tasks(self.template.tasks)
+            if not self.pretend:
+                self._execute_tasks(self.template.tasks)
```

Comment:
Maybe it makes more sense to add this logic directly into `_execute_tasks()`? So you just fix it in one place.

Reply:
I had thought about that actually, but my reasoning against it was that `_execute_tasks()` is agnostic of the kind of task it executes. In fact, it's just a thin wrapper around `subprocess.run()` which runs in the commands in the subproject's root folder. I wasn't sure whether I could justify disabling *any* (future) kind of task in pretend mode, that's why I added the condition in the several places explicitly.

That said, do you think it's safe to disable any kind of task execution?

Reply:
I think so. At some point we should let the user make an informed decision about whether executing each task or migration, to make copier safer. But that requires some refactoring. The shorter the code, the easier it will be to refactor when the time comes  IMHO.

Reply:
Okay, makes sense. I've moved the pretend mode check to the `_execute_tasks()` method. 39717b5